assembl/static2/js/app/graphql/mutations/createPost.graphql

```diff
@@ -1,18 +1,20 @@
 #import "../Post.graphql"
 
-mutation createPost($contentLocale: String!, $ideaId: ID!, $subject: String, $body: String!, $parentId: ID) {
-  createPost(ideaId: $ideaId, subject: $subject, body: $body, parentId: $parentId) {
+mutation createPost(
+  $contentLocale: String!
+  $ideaId: ID!
+  $subject: String
+  $body: String!
+  $parentId: ID
+  $attachments: [String]
+) {
+  createPost(ideaId: $ideaId, subject: $subject, body: $body, parentId: $parentId, attachments: $attachments) {
     post {
       ...Post
       ... on Post {
+        # we need to retrieve parentId and creationDate which are not fetched in Post fragment
```

Comment:
Is this so the apollo cache is updated with the new values?

Reply:
Yes. It's the createPost mutation, we don't have the info yet.